.github/workflows/build-pipeline.yml

```diff
@@ -160,15 +159,15 @@ jobs:
         run: |
           poetry run sphinx-build -b html ./docs docs/_build/
       - name: Publish Docs
-        uses: JamesIves/github-pages-deploy-action@4.1.5
+        uses: JamesIves/github-pages-deploy-action@v4
```

Comment:
were going backwards in version?

Reply:
Nope, `v4` is just an alias for "the latest in the 4.x series". They update the v4 tag when they release a new patch.